time_weighted_average_price/attest_fn_call/Makefile

```diff
@@ -15,32 +15,38 @@ tmp/x.wasm: main.go $(wildcard as/*.go) $(wildcard price/*.go) | tmp check
         -v .:/src \
         -w /src \
         tinygo/tinygo:0.31.2 \
-        tinygo build -o tmp/x.wasm -target=wasi main.go
+        tinygo build -o tmp/x.wasm -target=wasi ./...
 
 build: tmp/x.wasm
 
 init: build | check
 	@sed -e 's/VAR_EATTEST/null/' -e 's/VAR_TATTEST//' ./iteration-call.json.template > ./tmp/iteration-call.json
 	@cat tmp/iteration-call.json | bky-as attest-fn-call  | jq . > tmp/prev.json
-	@jq -r '.function_calls[0].claims.output | @base64d' tmp/prev.json | jq .
+	@jq -r '.transitive_attested_function_call.claims.output | @base64d' tmp/prev.json | jq .
 
 iteration: build | check
 	@sed \
 		-e 's/VAR_TATTEST/"tAttest": ${prev_ta},/' \
 		-e 's/VAR_EATTEST/${prev_ea}/' \
 		iteration-call.json.template > tmp/iteration-call.json
 	@cat tmp/iteration-call.json | bky-as attest-fn-call | jq . > tmp/prev.json
-	@jq -r '.function_calls[0].claims.output | @base64d | fromjson' tmp/prev.json
-	@jq -r '.function_calls[0].logs | @base64d' tmp/prev.json
+	@jq -r '.transitive_attested_function_call.claims.output | @base64d | fromjson' tmp/prev.json
+	@jq -r '.transitive_attested_function_call.logs | @base64d' tmp/prev.json
 
 twap: build | check
 	@sed \
 		-e 's/VAR_TATTEST/"tAttest": ${prev_ta},/' \
 		-e 's/VAR_EATTEST/${prev_ea}/' \
 		twap-call.json.template > tmp/twap-call.json
 	@cat tmp/twap-call.json | bky-as attest-fn-call | jq . > tmp/twap.json
-	@jq -r '.function_calls[0].claims.output | @base64d | fromjson' tmp/twap.json
-	@jq -r '.function_calls[0].logs | @base64d' tmp/twap.json
+	@jq -r '.transitive_attested_function_call.claims.output | @base64d | fromjson' tmp/twap.json
+	@jq -r '.transitive_attested_function_call.logs | @base64d' tmp/twap.json
+
+run-error: build | check
+	@echo "Running WASM module..."
+	@cat errorFunc.json | bky-as attest-fn-call >tmp/error.json
+	@echo "Output:"
+	@jq -r '.transitive_attested_function_call.claims.output | @base64d | fromjson' tmp/error.json
```

Comment:
Adding a way to generate an attestation with error, so that we can test the revert path in onchain/tests/user.ts